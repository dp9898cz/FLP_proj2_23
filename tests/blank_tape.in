S a B a
S a C b
C b F f
a a