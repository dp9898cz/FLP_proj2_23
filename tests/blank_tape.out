Sa a
Cb a
Ff a